.github/workflows/build_fork.yml

```diff
@@ -165,6 +135,12 @@ jobs:
           lean --version
           lake --version
 
+      - name: update {Mathlib, Tactic, Counterexamples, Archive}.lean
```

Comment:
Ok, that's weird.  There should be a CI step that recreates `Mathlib`, `Tactic`, `Counterexamples` and `Archive`: this is used to make sure that they get built and work.

This step was *not* introduced in this PR, but should be already present in regular CI.  This PR should simply shift to using `lake exe mk_all` for this, instead of the bash script.

Besides this, it is true that regular CI builds `Mathlib` once before this step.  I suspect that this is the step that was present before `Archive` and `Counterexamples` were added to be able to cache their oleans.  It is possible that there could be a single step that updates these "import all" files, but I would not investigate this here.

Reply:
@grunweg is this now considered "resolved" at least as far as this PR is concerned?

Reply:
Yes, it is: the check if all imports are present (for mathlib, archive, counterexamples and `Mathlib/Tactic`) is removed as a separate workflow step, but is now part of the build workflow (and runs before getting the cache or building mathlib). So, users still get early feedback, just via a different channel.